code/datums/outfits/jobs/cargo.dm

```diff
@@ -19,7 +19,7 @@
 
 /decl/hierarchy/outfit/job/cargo/mining
 	name = OUTFIT_JOB_NAME("Shaft miner")
-	uniform = /obj/item/clothing/under/rank/miner
+	uniform = /obj/item/clothing/under/rank/miner{ starting_accessories=list(/obj/item/clothing/accessory/storage/overalls) }
```

Comment:
Anonymous subtyping ("modified types" in dm ref) should be avoided where possible (ie, everywhere that is not in a dmm). It's very unclean for maintenance and creates hidden unique things.

Reply:
So what would be preferable? Creating a subtype of under/rank/miner (in this case) called like under/rank/miner/startgear that has the starting accessory gear defined, exclusively for using in the job outfit decls? I did it this was because there were a couple of preexisting outfits that used this quick tag-on.